src/dev/src/app/datagrid/pagination/pagination.html

```diff
@@ -81,7 +81,9 @@ <h2>Pagination</h2>
   </clr-dg-row>
 
   <clr-dg-footer>
-    {{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} of {{pagination.totalItems}} users
-    <clr-dg-pagination #pagination [clrDgPageSize]="10"></clr-dg-pagination>
+    <clr-dg-pagination #pagination [clrDgPageSize]="20">
+      <clr-dg-page-size [clrPageSizeOptions]="[10,20,50,100]">Users per page</clr-dg-page-size>
+      {{pagination.totalItems}} users
```

Comment:
Nitpick, but can you put back the:
```
{{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} of {{pagination.totalItems}} users
``` 
Or is that just not the recommended text anymore?

Reply:
Yes, we can do that. I was just following the sample text from the new pagination design spec.